server/routes/participant.js

```diff
@@ -424,13 +425,13 @@ employerActionsRouter.delete(
     // Get user
     const user = req.hcapUserInfo;
     // Get request body
-    const { body: { participantId, multiOrgHire } = {} } = req;
+    const { body: { participantId, multiOrgHire, currentStatusId } = {} } = req;
     if (!participantId) {
       return res.status(400).send('Missing participantId');
     }
 
     if (multiOrgHire) {
-      await removeAllParticipantStatusForUser({ user, participantId });
+      await removeAllParticipantStatusForUser({ participantId, currentStatusId });
```

Comment:
new definition of function only has { currentStatusId } in server/services/participants.js?